src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx

```diff
@@ -190,10 +199,12 @@ const NodeDetailComponent = ({
         const isTabFound = isResourceBrowserView
             ? markTabActiveByIdentifier(_idPrefix, params.node, params.nodeType, _url)
             : AppDetailsStore.markAppDetailsTabActiveByIdentifier(params.podName, params.nodeType, _url)
+        setSelectedTabName(_tabName)
 
         if (!isTabFound) {
             setTimeout(() => {
-                let _urlToCreate = `${url}/${_tabName.toLowerCase()}`
+                /* NOTE: shouldn't this be _url instead of url */
+                let _urlToCreate = _url
```

Comment:
it can be done. initially there was few case due to which it was written like this